### PR TITLE
Updated examples and extended the usage section

### PR DESCRIPTION
**Pull request by lukejacksonn**
Also forked and updated a codepen demo.

https://codepen.io/lukejacksonn/pen/xXRGpv?editors=0010

Let me know if you like any of it or want something changing!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,23 +8,26 @@
 [travis-badge]: https://travis-ci.org/picostyle/picostyle.svg
 [travis]: https://travis-ci.org/picostyle/picostyle
 
-<div align="center">
-  <a href="https://github.com/morishitter/picostyle">
-    <img width="360px" src="http://morishitter.github.io/picostyle.svg">
-  </a>
-</div>
-<br>
+<a href="https://github.com/morishitter/picostyle">
```
Comment (JorgeBucaran) [inline README.md]: @lukejacksonn Maybe remove the logo from the README? It's already visible on the org page.

**Comment by JorgeBucaran** [comment]
@lukejacksonn Can you follow the same code format used in the code, no semi and double quotes.

I left you a few comments too.

File: README.md [hunk 2]
```diff
@@ -8,23 +8,26 @@
...
+[Try it Online](https://codepen.io/lukejacksonn/pen/xXRGpv?editors=0010)
 
 ## Features
 
 - **🚀 The smallest CSS-in-JS library**: Only 0.3 KB (minified & gzipped).
 - **👏 Zero dependencies**: And under 60 LOC.
 - **💅 Styled components**: Gives you a styled component like [styled-components](https://www.styled-components.com/) that y'all love.
-- **❤️ For Hyperapp & Picodom**: The 1 KB frontend library family.
 
 ## Installation
```
Comment (JorgeBucaran) [inline README.md]: Maybe add a section to explain how to grab from unpkg, jsDelivr, etc. Just like we do in Hyperapp, etc.

File: README.md (hunk 1 above)
Comment (lukejacksonn) [reply README.md]: I agree. Have removed 👍

File: README.md (hunk 2 above)
Comment (lukejacksonn) [reply README.md]: Copied the one from hyperapp :+1:

**Comment by morishitter** [comment]
@lukejacksonn Thank you. Seems nice 👍

**Pull request merged**